controller/lib/core/sensors.h

```diff
@@ -28,6 +28,23 @@ Arduino Nano and the MPXV5004GP and MPXV7002DP pressure sensors.
 #include "pid.h"
 #include "units.h"
 
+struct SensorReadings {
+  Pressure patient_pressure;
+  // Pressure differences read at the inflow/outflow venturis.
+  Pressure inflow_pressure_diff;
+  Pressure outflow_pressure_diff;
+
+  // Inflow and outflow at the two venturis, calculated from inflow/outflow
+  // pressure diff.
+  VolumetricFlow inflow;
+  VolumetricFlow outflow;
+
```

Comment:
Since we also have corrected (total) flow, which is no longer inflow - outflow, I think it would make sense to add it to this structure.
I think this is something that would probably be useful for breath detection.

Reply:
Yeah, I agree.  But I don't think it's worth adding only because the whole reason I'm doing this is to move volume calculation and flow correction into Controller (see the TODO in this struct).  If we were to add it now we'd just need to remove it after that PR anyway.